### DISCUSSION TITLE
About "Packages" : Links for Rego Packages : Use cases

### DISCUSSION THREAD

**Discussion by sbalakrishn** [open-policy-agent/community#260] [Community]
Hi
Can some one provide me with examples of Packages that Rego have .
Is there a link to "Packages used in Rego" .

Explaining what packages we must use under what kind if use-cases.

**Comment by anderseknert** [comment]
Hi there! Are you talking about policy packages published by others? There's a list of them in the [Awesome OPA](https://github.com/anderseknert/awesome-opa#policy-packages) list. If you mean something else, perhaps you could be a bit more specific, and we'll hopefully be able to help you :)

**Comment by peteroneilljr** [comment]
Hi @sbalakrishn 

A package is a logical grouping of rules within your policy. A package name is required for each policy or set of rules, but the naming and division of rules are up to the author.

If you're making an app you might divide your rules based on function.

package security.authz
package security.net
package data.production
package data.development

If you're securing your AWS infrastructure you might want to group the rules based on resources.

package aws.compute.ec2
package aws.db.rds

You can read more about the package structure in our docs linked below. Let us know if you have any specific questions about how this works.

https://www.openpolicyagent.org/docs/latest/policy-language/#packages

Cheers,
Peter
